docs/requirements.txt

```diff
@@ -0,0 +1,10 @@
+numpy
+scipy
+matplotlib
+git+git://github.com/QENSlibrary/QENSmodels.git
+h5py
+nbsphinx
+jupyter
+bumps==0.7.12
```

Comment:
Is there a reason why docs/requirements.txt has a specific version of bumps but requirements.txt doesn't?

Reply:
`Bumps` is only required when using the Jupyter notebooks (in the docs).
These examples are run by `nbsphinx`. 
That's why bumps is in the docs/requirements (due to `nbsphinx`) and not in the requirements.txt which is in the main folder.